servicenow/sample-templates/xebialabs/USAGE.md.tmpl

```diff
@@ -0,0 +1,19 @@
+The blueprint supplies two yaml files with sample templates for usage with the ServiceNow application from the appstore.
+
+To deploy this blueprint with the XebiaLabs DevOps Platform, follow the steps below:
```

Comment:
Missing docker-compose part and also please mention that the user needs to mount a volume with the ServiceNow plugin in it

Reply:
It's explained here https://docs.xebialabs.com/xl-deploy/concept/docker-images-for-xl-deploy.html#plugins-directory-plugins

Reply:
Added a section that informs the user about the plugib and how to do that.